projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -104,7 +104,7 @@ function () {
 				My_Jetpack_Initializer::init();
 			}
 		);
-		add_action( 'init', array( new Automattic\Jetpack\Social\Note(), 'register' ) );
+		add_action( 'init', array( new Automattic\Jetpack\Social\Note(), 'init' ) );
```

Comment:
I've noticed that all the methods are static in Note, but this is creating an instance. We might want to reconfigure that a bit, and maybe expose the instance somewhere so people can add/remove action hooks easily.

Reply:
I have made the methods non-static, @pablinos. Do you think it's better now? One could initialize the note class to do what they want.

Reply:
Yeah I think that's good. I was wondering whether we should store the instance of `Note` somewhere, so if someone wants to remove a hook or filter, then they can. I was thinking that we could perhaps make it a public property of the Jetpack_Social class, but we don't make [that instance](https://github.com/Automattic/jetpack/blob/trunk/projects/plugins/social/jetpack-social.php#L91-L92) easily accessible either.

Perhaps we should sort this out in a different PR!